### ISSUE TITLE
Parallel compilation of Gleam

### ISSUE THREAD

**Issue by lpil** [gleam-lang/gleam#156]
We already compile the generated Erlang in parallel, but it would be nice to do the same for Gleam code.

**Comment by lpil** [comment]
Closing as we need to tackle incremental computation first.

**Comment by lpil** [comment]
Closing in favour of more fine grained issues
